cmd/snap/cmd_advise.go

```diff
@@ -60,6 +64,7 @@ func init() {
 		return &cmdAdviseSnap{}
 	}, map[string]string{
 		"command":  i18n.G("Advise on snaps that provide the given command"),
+		"dump-db":  i18n.G("Dump advise database for use by command-not-found."),
```

Comment:
"Dump entire advise database to standard output"

The idea is not assuming a given use case.